Buildings/Experimental/DHC/Loads/Steam/BaseClasses/ValveSelfActing.mo

```diff
@@ -31,17 +31,20 @@ model ValveSelfActing "Ideal pressure reducing valve for steam heating systems"
   Modelica.Blocks.Sources.Constant zer(final k=0) "Zero"
     annotation (Placement(transformation(extent={{-40,70},{-20,90}})));
 equation
+  assert(dpReq.y < 0, "pb_nominal is set higher than the upstream pressure 
+  (results in negative pressure drop)", AssertionLevel.warning);
+
   connect(port_a, ideSou.port_a)
-    annotation (Line(points={{-100,0},{40,0}},color={0,127,255}));
+    annotation (Line(points={{-100,0},{40,0}}, color={0,127,255}));
   connect(ideSou.port_b, port_b)
     annotation (Line(points={{60,0},{100,0}}, color={0,127,255}));
-  connect(pbSet.y, add.u1)
+  connect(pbSet.y,dpReq. u1)
```

Comment:
Please remove space after the "." so that you have `connect(pbSet.y,dpReq.u1)`

Reply:
done